types/react-autocomplete/react-autocomplete-tests.tsx

```diff
@@ -9,6 +9,7 @@ const items = ["hello", "world"];
     getItemValue={(item) => item}
     renderItem={(item) => <div key={item}>{item}</div>}
     value={items[0]}
+    autoHighlight={false}
```

Comment:
huh this actually [errors](https://github.com/DefinitelyTyped/DefinitelyTyped/actions/runs/9427647096/job/25972055110?pr=69761)... have I fundamentally misunderstood how defaults work for interfaces? 🤔

Reply:
Switched to JSDoc `@default` in that case